python/nav/models/profiles.py

```diff
@@ -199,6 +201,14 @@ def is_admin(self):
         """Has this user administrator rights?"""
         return self.has_perm(None, None)
 
+    @property
+    def is_anonymous(self):
+        return self.id == self.DEFAULT_ACCOUNT
+
+    @property
+    def is_authenticated(self):
+        return self.id != self.DEFAULT_ACCOUNT
+
```

Comment:
I would recommend `is_staff = False; is_superuser = False` when testing out 3rd party libraries depending on `django.contrib.auth`.

Reply:
Eh, I guess you could make 'em depend on `is_admin` somehow.

Reply:
I've added these as properties to the Account class in the latest update.